source/javascripts/gradient_fade.js

```diff
@@ -0,0 +1,16 @@
+function init() {
+    // Create the listener function
+    var updateLayout = _.debounce(function(e) {
```

Comment:
An OK solution is found here: http://folio.artsy.net / https://github.com/artsy/folio.artsy.net/blob/master/source/javascripts/application.coffee

Also I noticed waypoints elsewhere—just do this with a waypoint?